phpcs.xml.dist

```diff
@@ -64,6 +64,9 @@
 		<exclude name="WordPress.NamingConventions.ValidVariableName"/>
 		<exclude name="WordPress.WP.Capabilities.Undetermined" />
 
+		<!-- Include only once we support PHP 8.3 and above -->
+		<exclude name="SlevomatCodingStandard.TypeHints.ClassConstantTypeHint" />
```

Comment:
@justlevine  - Commenting for myself so I remember to update wpgraphql-coding-standards